UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Web/Services/NMDataService.cs

```diff
@@ -188,31 +188,64 @@ public async Task<YearWeekResponseDataModel> GetAllYearWeek(string correlationId
             return new YearWeekResponseDataModel { YearWeekModel = yearWeekModelList, IsYearAndWeekAttributesCached = isCached };
         }
 
-        public async Task<List<ShowDailyFilesResponseModel>> GetDailyBatchDetailsFiles(string correlationId)
+        public async Task<ShowDailyFilesResponseListModel> GetDailyBatchDetailsFiles(string correlationId)
```

Comment:
There's a lot of shared code between this method and  GetWeeklyBatchFiles (and possibly getAllYearWeek) - this would be a good candidate for a refactor.

However, given time limits (and one more of these change due for cumulatives), possibly not the best time to start one.